hare/hare.go

```diff
@@ -373,8 +373,6 @@ func (h *Hare) getGoodProposal(lyrID types.LayerID, epochBeacon types.Beacon, lo
 	if err != nil {
 		if err != database.ErrNotFound {
 			logger.With().Error("no proposals found for hare, using empty set", log.Err(err))
-		} else if err = h.mesh.SetZeroBallotLayer(lyrID); err != nil {
```

Comment:
do you know why we don't need this now? i recall i added this due to tortoise test failures.

Reply:
can't tell anything about tortoise failures. in general we only need to know if hare_output is empty or nil, other than that tortoise doesn't care if layer has any ballots/blocks